packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA - DR App Interface.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8"?><channel version="4.4.2">
+  <id>8240f77a-19d4-4225-90fc-ed1c232286c3</id>
+  <nextMetaDataId>3</nextMetaDataId>
+  <name>XCA - DR App Interface</name>
+  <description>XCA App Interface channel
+ - forwards a single request back to the calling app or DB
+
+Last updated: Nov 30 2023</description>
```

Comment:
is the `Last updated` set by us manually? If so, let's just remove this? We won't remember to always keep this up to date